docs/quickstart.md

```diff
@@ -33,61 +33,55 @@ For more detailed installation requirements, please refer to [installation](/dep
 
-## Add a Kubernetes Cluster as Deployment Target
+3. After logging in, you can set a new password and configure the access address of Walrus.
 
-> Prerequisites:
->
-> - A Kubernetes cluster that can be reached by the Walrus server.
+![qs-set-pwd](/img/v0.5.0/quickstart/qs-set-pwd-en.png)
+
+## Local Environment
 
-1. Click the `Application Management` menu in the navigation bar to go to the `default` project view by default.
-2. Click the `Connectors` tab.
-3. Click the `New Connector` button, and select `Kubernetes` type.
-4. Fill in the cluster name and KubeConfig file, and click save.
+Walrus provides a default `local` environment under the `default` project for you to quickly experience the application deployment process. 
 
-![qs-add-connector](/img/v0.4.0/quickstart/qs-add-connector-en.png)
+![qs-local-env](/img/v0.5.0/quickstart/qs-local-env-en.png)
 
-## Create Environment
+The `local` environment connects to a local Kubernetes cluster connector created by built-in K3s. You can use it to deploy applications without having to configure a Kubernetes cluster.
 
-1. Click the `Environment` tab, and click the `New Environment` button.
+![qs-local-connector](/img/v0.5.0/quickstart/qs-local-connector-en.png)
 
-![qs-create-env](/img/v0.4.0/quickstart/qs-create-env-en.png)
+## Deploy Resources
 
-2. Enter the environment name, such as `dev`.
-3. Click `Add Connector` button, and select the Kubernetes connector added in the previous step.
-4. Click save.
+1. In the environment list page, click the `local` environment to enter the environment view.
 
-![qs-create-env2](/img/v0.4.0/quickstart/qs-create-env2-en.png)
+![qs-env-list](/img/v0.5.0/quickstart/qs-env-list-en.png)
 
-## Deploy Service
+2. Click the `New Resource` button to enter the resource creation page.
 
-1. In the environment list page, click the environment name created in the previous step to enter the environment view.
+![qs-create-res](/img/v0.5.0/quickstart/qs-create-res-en.png)
 
-![qs-go2env](/img/v0.4.0/quickstart/qs-go2env-en.png)
+3. Fill in the resource name, such as `myapp`, and make sure the resource type is `containerservice`.
 
-2. Click the `New` button and select `Service`.
+4. Enter `nginx` image in the Image Name of the template configuration, fill in the port mapping, and click the `Save` button.
 
-![qs-create-svc](/img/v0.4.0/quickstart/qs-create-svc-en.png)
+![qs-create-res-nginx](/img/v0.5.0/quickstart/qs-create-res-nginx-en.png)
 
-3. Fill in the service name, such as `myapp`, and make sure the template is selected as `kubernetes-containerservice`.
-4. Enter `nginx` image in the Image Name of the template configuration, and click confirm.
+5. Click the `Save and Deploy` button to complete the creation of the resource.
+6. After the resource deployment is complete, you can view components, logs, execute terminal commands, access the address of this nginx service, etc.
 
-![qs-create-svc2](/img/v0.4.0/quickstart/qs-create-svc2-en.png)
+- Access the endpoint.
```

Comment:
Add `-p 30000-30100:30000-30100` to the install command to make it work. Also, add a note to explain it.